src/server_manager/web_app/app.spec.ts

```diff
@@ -56,7 +53,7 @@ describe('App', () => {
   it('creates a manual server with valid input', async () => {
     // Create a new app with no existing servers or DigitalOcean token.
     const appRoot = document.getElementById('appRoot') as unknown as AppRoot;
-    const app = createTestApp(appRoot, new InMemoryDigitalOceanTokenManager());
+    const app = createTestApp(appRoot, createCloudAccounts());
```

Comment:
Replace all `createCloudAccounts()` with no parameters with `new CloudAccounts(new FakeDigitalOceanAccount(), new InMemoryStorage())`.

I find the dual behavior of the existing method confusing. Or provide two separate methods. Perhaps something like makeCloudAccountsWithEmptyDo() and makeCloudAccountsWithDoAccount().

Reply:
Good call. I replaced the `createCloudAccounts` with `makeCloudAccountsWithDoAccount` and inlined the ones that don't use a fake account.